### DISCUSSION TITLE
Mainsail connected, Kilipper disconnected

### DISCUSSION THREAD

**Discussion by Flashjunky** [bassamanator/Sovol-SV06-firmware#149] [Q&A] [answered]
I'm using a BTT Pad 7 with an SV06 Plus stock printer.

I've followed the guide (I even agree about Robert Redford!) and updated all the components from the BTT pad, but I don't even know where to start to be able to work out where my ignorance ends or my stupidity starts.

So, where can I look for logs etc. to work out why Klipper is disconnected?

Thanks in advance!!

**Comment by bassamanator** [comment]
Well, the mainsail dashboard itself will tell you in red what the issue is, that's the first place to look. How about a picture?

**Reply by Flashjunky** [reply]
![IMG_0168](https://github.com/bassamanator/Sovol-SV06-firmware/assets/39931199/962de14f-1799-4b2b-99c3-e8054b935b99)

**Comment by bassamanator** [comment]
Let's see the error in the console, and logs are in `~/printer_data/logs`.

**Reply by Flashjunky** [reply]
I'm sorry if I appear dumb, but I don't know what you mean.  I cannot find any console to see an error in, and the log files in that folder are all from 2023.  I'm not a computer dummy but I am a Linux dummy for sure!

**Comment by Flashjunky** [comment, marked as answer]
omg.  SO stupidly simple.  the tape for the 5v pin had moved and was partially covering one of the other pins.

IT'S ALIIIIVE!!!!

Thank you for tolerating my idiocy :)

**Comment by bassamanator** [comment]
No worries at all. 👍🏼
